models/intel/face-landmarks-98-detection-0001/model.yml

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2021 Intel Corporation
```

Comment:
not sure that this file should be here (model will be uploaded from internal repo to s3), please remove

Reply:
I'm confirm that, for Intel models we take model.yml file from models package, once it uploaded to S3 storage